src/features/click-to-load.js

```diff
@@ -1615,6 +1616,10 @@ export default class ClickToLoad extends ContentFeature {
         const localizedConfig = getConfig(locale)
         config = localizedConfig.config
         sharedStrings = localizedConfig.sharedStrings
+        // update styles if asset config was sent
+        if (args?.assets) {
+            styles = getStyles(args.assets)
```

Comment:
Can we expose these on the class somehow? I'm trying to deprecate `args` lately.